### ISSUE TITLE
Request to move all repos from jupyter-incubator into jupyter

### ISSUE THREAD

**Issue by Carreau** [jupyter/software-steering-council-team-compass#24]
This mirrors https://github.com/jupyter-governance/ec-team-compass/issues/51, 

There are 4 repos in juyter incubator:

- Spark magic (last activity 6 month ago), move to jupyter as is
- proposals, contentmanagement, declarativewidgets_explorer (last activity 4+ years ago, and archived) move to Jupyter as well.

Then archive the org and kick everybody out.

This is in order to start reducing the number of orgs, as managing many orgs is a hassle for many reasons: managing security reports, managing teams, GitHub sponsors, blocking users org wide ... etc.

**Comment by jtpio** [comment]
cc @jupyter/software-steering-council for awareness

**Comment by Carreau** [comment]
SSC should have met yesterday and on sept 23rd, is there any update @jupyter/software-steering-council ?

**Comment by vidartf** [comment]
@Carreau Notes from previous call:

> Current incubator process documented here:
> https://docs.jupyter.org/en/latest/projects/incubator.html 
> 
> Next steps:
> 
> - Update documentation on how to bring new projects to Jupyter depending on the project (rules of thumb)
>   - New subproject (e.g. Jupyter-Book)
>   - New repository in existing subproject
>   - Potentially: mention contrib / Third-party repositories? Should they be "officially acknowledged" by Jupyter without being part of Jupyter.
> - Update governance repository to remove links to attic and incubator - and add link to the documentation mentioned in the previous bullet point.
> - Archive jupyter-incubator repository.
>   - Move spark magic to Jupyter or IPython.
>   - Move other repositories under Jupyter under the archived state.


tl;dr: In addition to the steps you outlined, we should make sure the docs on jupyter.org are updated from mentioning incubator to mentioning current ways to on-board new projects/repos. The governance text also needs updating as it currently explicitly mentions incubation multiple places, some places with references to the current process:
- https://github.com/jupyter/governance/blob/f8239810abfe18b80a09f45809a5b934296e78ba/software_steering_council.md?plain=1#L18
- https://github.com/jupyter/governance/blob/f8239810abfe18b80a09f45809a5b934296e78ba/software_subprojects.md?plain=1#L18-L22

**Comment by vidartf** [comment]
PS: I don't think a larger vote is need, as I think the SSC will still own the "incubation process", it is just that some tweaks to the governance text would be needed to make it clearer that the SSC is deemed in charge of how incubation is done, but without an explicit link to the incubator org.

**Comment by Carreau** [comment]
Thanks.

**Comment by krassowski** [comment]
I opened a draft https://github.com/jupyter/governance/pull/292.

I will just note that in void of incubator there are now two or more orgs which functionally fulfill the role of incubator:
- `jupyter-ai-contrib`
- `jupyterlab-contrib`

Therefore, I am really unconvinced if not having an incubator is the best idea as having many unaudited incubatos does not feel more secure at all to me. If there is a strong push against having an incubator or, should we document that incubation typically happens in unofficial - contrib organizations and link them?

**Comment by Carreau** [comment]
there is no reasons that an incubating project not be under an existing jupyter org.
